src/index.ts

```diff
@@ -9,8 +9,8 @@ import * as YAML from 'js-yaml'
 
 const token = process.env.MESG_TOKEN;
 const ymlPath = './mesg.yml';
-const endpoint = process.env.MESG_ENDPOINT || 'localhost:50052';
-const endpointTCP = process.env.MESG_ENDPOINT_TCP || '';
+const defaultEndpoint = process.env.MESG_ENDPOINT || 'localhost:50052';
+const defaultEndpointTCP = process.env.MESG_ENDPOINT_TCP || '';
```

Comment:
`MESG_ENDPOINT_TCP ` has been deprecated for a few month now. You can only use `MESG_ENDPOINT` in service.

Reply:
done